packages/block-library/src/shortcode/edit.js

```diff
@@ -23,6 +23,7 @@ export default function ShortcodeEdit( { attributes, setAttributes } ) {
 				className="blocks-shortcode__textarea"
 				id={ inputId }
 				value={ attributes.text }
+				aria-label={ __( 'Shortcode text' ) }
```

Comment:
"text" might be not the best fit for shortcodes. How about 'Shortcode content'?

Reply:
"Content" sounds like it might refer to stuff between shortcode tags, but not including them, so I'm not sure if it's the right term. What do you think, @afercia?